source/StarField.cpp

```diff
@@ -180,13 +187,13 @@ void StarField::Draw(const Point &pos, const Point &vel, double zoom) const
 				int index = (gx & widthMod) / TILE_SIZE + ((gy & widthMod) / TILE_SIZE) * tileCols;
 				int first = 6 * tileIndex[index];
 				int count = 6 * tileIndex[index + 1] - first;
-				glDrawArrays(GL_TRIANGLES, first, count);
+				glDrawArrays(GL_TRIANGLES, first, count / pow(j, 2.));
```

Comment:
```suggestion
				glDrawArrays(GL_TRIANGLES, first, count / pow(passes, 2.));
```
what does pow even do?